cla_public/templates/base.html

```diff
@@ -51,32 +54,46 @@
 
 {% block head %}
   <meta http-equiv="content-language" content="{{ request.cookies.get('locale', 'en')[:2] }}">
+{%- endblock %}
```

Comment:
This looks like a typo and I'm confused why it doesn't break something!

Oh, just seen that there was already one of these in the template lower down. Is it a thing? Or silently ignored? Or is our template structure not what we think it is?

Reply:
`{%-` is apparently for controlling vertical space https://stackoverflow.com/questions/11813422/what-does-this-in-jinja2-template-engine-do
https://tedboy.github.io/jinja2/templ6.html

That was already there, I think it looks like an addition due to how the diff is generated

Reply:
Ah ok, didn't realise that was a thing. Cool. Thanks